packages/universe-components/src/donationForm/DonationForm.tsx

```
@@ -137,38 +147,44 @@ export default function DonationForm({ initiative, rate }: DonationFormProps) {
-          //});
+            //});
+          }
+          //console.log('CHAIN2', chainState)
         }
-        //console.log('CHAIN2', chainState)
-      }
-    });
+      });
   }
 
   useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: getRate

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)